pkg/session_recording/recordings.jsx

```diff
@@ -340,7 +349,7 @@
          * Set the cockpit location to point to the specified recording.
          */
         navigateToRecording(recording) {
-            cockpit.location.go([recording.id]);
+            cockpit.location.go([], $.extend(cockpit.location.options, { id: recording.id }));
```

Comment:
Hmm, I think having the recording ID in the "path" kind of made sense, since it's the actual object we're showing, and the rest are just parameters which only apply to the `/session_recording/` location. Is there something I'm missing, or perhaps you had some other idea?

Reply:
I've forgot to mention - I have not found a nice and proper way to form such URL with cockpit.location for now.

Reply:
Argh. What did you try and what did you get?

Reply:
It seems that cockpit.location is only setting parameters after # basically. But this can be done just using window.location.
P.S. It was not only thing I was working on.

Reply:
Sure, no problem. Could you show how it looks, paste a link?

Reply:
Right I have only current version - same as in pull request. window.location ( default JS obj ) stores this - "http://localhost:9090/cockpit/@localhost/session_recording/index.html#/"

Reply:
Right, as an old-timer I didn't expect that :) I think just putting `[recording.id]` into the first argument of `cockpit.location.go` would be fine. So the URL would be: http://localhost:9090/session_recording#/0777dde78ccb4265862ab18d1857bbfd-590-41522?username=user3&dateSince=2017-08-01&dateUntil=2017-08-31

Reply:
OTOH, if we add host IDs or names to this (as in #20), then it becomes more complicated. Perhaps we should stick with just parameters, as you made it.

Reply:
Well, yeah, options might be more reliable, but I just made it with path[0] for ID. If you want I can revert it, what do you think? We can do it later and for now keep it as is.

Reply:
Yeah, let's leave it as is. We can change it later. And we can do this after all later: http://localhost:9090/session_recording#/HOST_ID/0777dde78ccb4265862ab18d1857bbfd-590-41522?username=user3&dateSince=2017-08-01&dateUntil=2017-08-31

Reply:
Aha! I see, agree. This is really interesting. We will get to this soon also.
What do you think in general about PR?